tests/h/emails/reply_notification_test.py

```diff
@@ -162,3 +199,7 @@ def token_serializer(self, pyramid_config):
         serializer.dumps.return_value = 'FAKETOKEN'
         pyramid_config.registry.notification_serializer = serializer
         return serializer
+
+    @pytest.fixture
+    def links(self, patch):
```

Comment:
Can we please try and keep fixtures alphabetically organised so they're easy to find?